internal/resources/tanzupackagerepository/resource_package_repository_test.go

```diff
@@ -80,7 +80,7 @@ func TestAcceptanceForPackageRepositoryResource(t *testing.T) {
 		}
 	}
 
-	t.Log("start packagerepository resource acceptance tests!")
+	t.Log("start packagerepository resource acceptance tests!", testConfig.ScopeHelperResources.Cluster.Name)
```

Comment:
Is it formatted correctly?

Reply:
yes, I've removed the printing of cluster name this was just for testing purpose